webpack.common.js

```diff
@@ -77,163 +78,177 @@ console.log('npm run watch  --themes_dir=themes/mytheme/client --css_file=myfile
+        options: {
+          name: '[name].[ext]',
+          outputPath: IMG_DIR_CONFIG,
+          compress: {
+            // This will take more time and get smaller images.
+            mode: 'low', // 'lossless', 'high', 'low'
+            disableOnDevelopment: true,
+            // convert to webp
+            webp: false,
+            // loseless compression for png
+            optipng: {
+              optimizationLevel: 4
             },
-            {
-              loader: 'resolve-url-loader'
+            // lossy compression for png. This will generate smaller file than optipng.
+            pngquant: {
+              quality: [0.2, 0.8]
             },
-            {
-              loader: 'postcss-loader',
-              options: {
-                postcssOptions: {
-                  plugins: ['autoprefixer']
-                }
-              }
+            // Compression for svg.
+            svgo: true,
+            // Compression for gif.
+            gifsicle: {
+              optimizationLevel: 3
             },
-            {
-              loader: 'sass-loader',
-              options: {
-                sourceMap: false
-              }
+            // Compression for jpg.
+            mozjpeg: {
+              progressive: true,
+              quality: 60
             }
-          ]
-        },
-        // images
-        {
-          test: /\.(png|svg|jpe?g|gif)$/,
-          loader: 'url-loader',
-          options: {
-            limit: 4096, // in bytes
-            outputPath: IMG_DIR_CONFIG,
-            name: '[name].[ext]'
+          },
+          inline: {
+            limit: 1
           }
-        },
-        // fonts
-        {
-          test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
-          use: [
-            {
-              loader: 'file-loader',
-              options: {
-                outputPath: FONTS_DIR_CONFIG,
-                name: '[name].[ext]'
-              }
-            }
-          ]
-        },
-        // add jQuery
-        {
-          test: require.resolve('jquery'),
-          use: [
-            {
-              loader: 'expose-loader',
-              options: {
-                exposes: ['$', 'jQuery']
-              }
-            }
-          ]
         }
-        // {
-        //     test: /\.svg$/i,
-        //     use: 'svg-inline-loader'
-        // },
-      ]
+      }]
     },
+    {
+      test: /\.(woff(2)?|ttf|eot)(\?v=\d+\.\d+\.\d+)?$/,
+      use: [{
+        loader: 'file-loader',
+        options: {
+          outputPath: FONTS_DIR_CONFIG,
+          name: '[name].[ext]'
+        }
+      }]
+    }
+    ]
+  },
 
-    // extra settings
-    resolve: {
+  // extra settings
+  resolve: {
 
-      // node modules to include
-      modules: [
-        path.join(__dirname, 'node_modules'),
-        path.resolve(NODE_DIR)
-      ],
+    // defines root folders for compatibility
+    roots: [
+      path.resolve('./../../'),
```

Comment:
That's an option to require files from the website root for example:
background: url(/vendor/silverstripe/admin/client/dist/images/chosen-sprite.png)